### PR TITLE
Cgi method

### PR DESCRIPTION
**Pull request by yann1731**
It's coming along

### CODE REVIEW

File: src/parsing.cpp
Comment (francislah) [inline src/parsing.cpp:1]: validateFirstToken and validateLastToken needs to be "cgi"

File: html/cgi-bin/api.py
Comment (francislah) [inline html/cgi-bin/api.py:1]: still necessary?

**Review by francislah — CHANGES_REQUESTED** [review]

File: html/cgi-bin/api.py
Comment (yann1731) [reply html/cgi-bin/api.py:1]: What do you mean?

File: src/parsing.cpp
Comment (yann1731) [reply src/parsing.cpp:1]: Fixed

**Review by francislah — APPROVED** [review]

**Pull request merged**